### PR TITLE
Add some docs

### PR DESCRIPTION
**Pull request by k0va1**
Hi,
While I was using gem it was difficult to figure out how it works at first glance. So I decided to put together all the stuff. Any suggestions and comments are welcome

### CODE REVIEW

File: docsite/source/index.html.md [hunk 1]
```diff
@@ -5,4 +5,149 @@ chapter: Factory
 
 # rom-factory
 
-TODO: write docs :)
+`rom-factory` provides simple API for creating and persisting ROM::Struct's. If you already know FactoryBot you'll definetily understand the concept.
```
Comment (solnic) [inline docsite/source/index.html.md:8]:
```suggestion
`rom-factory` provides a simple API for creating and persisting `ROM::Struct`'s. If you already know FactoryBot you'll definitely understand the concept.
```

File: docsite/source/index.html.md [hunk 2]
```diff
@@ -5,4 +5,149 @@ chapter: Factory
...
+
+## Installation
+
+First of all you need to define a ROM Container for Factory. For example if you are using `rspec`, you can put these lines to `spec_helper.rb`. Also you need to require here all files with Factories.
```
Comment (solnic) [inline docsite/source/index.html.md:12]:
```suggestion
First of all you need to define a ROM Container for Factory. For example if you are using `rspec`, you can add these lines to `spec_helper.rb`. Also you need to require here all files with Factories.
```

**Comment by k0va1** [comment]
Thanks! I hope now it's OK :)

**Comment by solnic** [comment]
@k0va1 thanks again! we've got docs now :) https://rom-rb.org/learn/factory/0.10/ it's funny because *yesterday* I looked at this page thinking "oh man, I gotta write these docs" hah...

**Comment by k0va1** [comment]
@solnic you're welcome) BTW, I've just checked the page https://rom-rb.org/learn/factory/0.10/#associations and came across a bug in `associations` section. The easiest way to fix it will be remove unordered list. But maybe it's a bug in theme

**Comment by solnic** [comment]
@k0va1 this looks like a mistake in md syntax

**Comment by k0va1** [comment]
@solnic hm, it looks OK here https://github.com/rom-rb/rom-factory/pull/58/files?short_path=00ae0ee#diff-00ae0ee4ccfb1882896653c6c718be6c

**Comment by solnic** [comment]
@k0va1 I fixed it. It's because "```" needs an empty line. The MD parser the website uses is not the same as GH, so looking at GH formatting is not enough. Unfortunately we don't have (yet) a preview feature, but in the future it will be possible to preview a PR on the actual website (it's possible to automatically build and deploy a test version for a specific PR).

**Comment by k0va1** [comment]
@solnic cool! Thanks for explaining the details :)

**Pull request merged**